death_clock_site/death_clock_app/forms.py

```diff
@@ -1,38 +1,63 @@
 from django import forms
-
 from django.core.exceptions import ValidationError
 from django.utils.translation import ugettext_lazy as _
-
+from .models import *
+
+# Question object for easy storage of form questions
+class QuestionObj:
+	text = ''
+	answers = []
+	def __init__(self, q, a):
+		self.text = q
+		self.answers = a
+
+# Given a question, returns all answers for the question
+def GetQuestions(q):
+	answerList = []
+	index = 0
+	for ans in Answer.objects.filter(question=q):
+		answerList.append((index, ans.text))
+		index += 1
+	return answerList
+
+# Question form
+class QuestionForm(forms.Form):
+	def __init__(self, *args, **kwargs):
+		super(QuestionForm, self).__init__(*args, **kwargs)
+
+		for q in Question.objects.all():
+			tempq = QuestionObj(q.question, GetQuestions(q))
+			self.fields[tempq.text] = forms.ChoiceField(label=tempq.text, choices=tempq.answers, required=True)
 
 class LoginForm(forms.Form):
-    username = forms.CharField(required=True, label='Username')
-    password = forms.CharField(widget=forms.PasswordInput(), required=True, label='Password')
+	username = forms.CharField(required=True, label='Username')
```

Comment:
how many spaces are your tabs? we should probably make a standard number of spaces so that we're not getting any whitespace errors.

Reply:
It seems according to PEP 8 it should be "four spaces per indentation level" 
[link](https://www.python.org/dev/peps/pep-0008/#id17)

Reply:
The function for getting answers is called GetQuestions. Probably just a simple mistake I'm assuming.

Reply:
Looks good now!!!